File: Python/summation.py
sum = 0
panjang = int(input("Masukkan jumlah angka yang ingin dijumlahkan: "))

temp = [None] * panjang
print(len(temp))
for i in range(0, panjang):
    temp[i] = int(input("Angka ke-" + str(i + 1) + ": "))
    sum += temp[i]
print(sum)

File: C++/summation.cpp
#include <iostream>

using namespace std;

int main()
{
    int panjang;
    int sum = 0;

    cout << "Masukkan jumlah angka yang ingin dijumlahkan: ";
    cin >> panjang;
    
    int temp[panjang];

    for (int i = 0; i < panjang; i++)
    {
        cout << "Angka ke-" << i + 1 << ": ";
        cin >> temp[i];
        sum += temp[i];
    }
    cout << sum << endl;
}